arrow-core/src/main/kotlin/arrow/core/extensions/validated.kt

```diff
@@ -185,3 +188,6 @@ interface ValidatedHash<L, R> : Hash<Validated<L, R>>, ValidatedEq<L, R> {
     HR().run { it.hash() }
   })
 }
+
+fun <E> Validated.Companion.nelApplicative(): ValidatedApplicative<NonEmptyList<E>> =
```

Comment:
Shouldn't it be `applicativeNel`, inline with `ValidatedNel`? That way people will also get this auto suggested if they start typing `Validated.app..`.

Reply:
derp, good point! 😄